spec/controllers/hyrax/batch_edits_controller_spec.rb

```diff
@@ -114,19 +133,23 @@
         allow(controller).to receive(:current_user).and_return(user2)
       end
 
-      it "fails to delete collections when user does not have edit access" do
-        controller.batch = [collection1.id, collection3.id]
+      it "fails to delete collections, and admin sets when user does not have edit access" do
```

Comment:
Should this be separate test cases?  With one test I'm not sure we can be certain both cases are covered.